pkg/scalers/artemis_scaler.go

```diff
@@ -208,6 +193,44 @@ func getAPIParameters(meta artemisMetadata) (artemisMetadata, error) {
 	return meta, nil
 }
 
+func parseArtemisAuth(config *ScalerConfig) (*authentication.AuthMeta, error) {
```

Comment:
As you have changed to `AuthMeta`, there is a helper to parse it `GetAuthConfigs` with all the options (event the scaler doesn't support all, we can use the helper and ignore invalid options)